posts/atomic-design-bootstrap.md

```diff
@@ -0,0 +1,154 @@
+_Bootstrap grid system example from the website_
+
+One of these elements is the [grid system](https://getbootstrap.com/docs/4.4/layout/grid), because it allow a grid simulation with 12 columns an manage some responsive design issues. Even if gutters (space between columns: 30px by default) may have a lot of effects on the building of a grid, they can be removed using `.no-gutters` modifier.
+
+Related to `Components`, the gris has a less impact on a perceptive and functional value. It is used to structure elements inside columns and can be used on the `organism` part to manage `molecules` and `atoms`.
+
+## Utilities
+
+Some elements are use as utilities, they usually have a lower graphical value than `Components` and are used by composition..
+
+As an example, we will use [text](https://getbootstrap.com/docs/4.4/utilities/text).
+
+This utility let you do generics actions like:
+
+* alignment
+* prevent line return and truncate
+* transform to lowercase, uppercase …
+* change weight and use italic
+* change font style to monospace
+
+Some rules are more generic than others: where alignment is common and will have a small effect on style, the specific font choice for something like monospace may have more impacts:
+
+```css
+.text-monospace {
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
+}
+```
+
+May be you can think it is not an important effect, but, depending on the utilities, this type of effect may vary, this is the point of [colors](https://getbootstrap.com/docs/4.4/utilities/colors) which has its own semantic and refers to _Bootstrap_ branding.
+
+You must be pragmatic to choose utilities.
+
+# Go further with Sass
+
+Extends and customize _Bootstrap_ can be done using  [Sass (SCSS)](https://github.com/twbs/bootstrap) [sources](https://sass-lang.com). It is possible to change container size depending on window size, colors, and there is many other exposed variables using `!default` Sass principle. Moreover, file fragmentation let you use parts of _Bootstrap_ like use only the grid system for example by importing the Sass file you want to use.
+
+In this case, you can use them for:
+
+* `Components` : Change colors for perception
+* `grid system` : Change columns number to fit what you need
+
+More information on what is available to extends _Bootstrap_ are on the [Theming Bootstrap](https://getbootstrap.com/docs/4.4/getting-started/theming) page.
+
+I recommend you to not override too many properties of _Bootstrap_ because your coupling can impact your maintainability and version upgrades.
+
+# Conclusion
+
+There is a lot of way to use _Bootstrap_ in an _Atomic Design_ project. With the experience and depending on the context, we learn how to not override, not use build-in _Bootstrap_ components without think about how they will be used and how they are perceived. Sometimes it is not needed to add the framework because it may not add value.
```

Comment:
There is a lot of ways*

we learn how not to override, not to use built-in _Bootsrap_ components without thinking about how they will be used and how they are perceived.*

Sometimes it is not needed to add to the framework*

Reply:
I just re-read this, maybe I missunderstood ! 

If you meant "we learn how not to override, nor use built-in _Bootsrap_ components...." do take my review into account. (Except you have to revert **to** and **not**)